de-lib/src/main/java/org/iplantc/de/client/services/impl/DiskResourceServiceFacadeImpl.java

```diff
@@ -1041,6 +1042,26 @@ public String downloadTemplate(String templateid) {
         return address;
     }
 
+    @Override
+    public void requestHTPathlistFile(HTPathListRequest request, DECallback<String> callback) {
+        StringBuilder address =
+                new StringBuilder(deProperties.getDataMgmtBaseUrl() + "path-list-creator?");
+        address.append("dest=" + URL.encodeQueryString(request.getDest()));
+        address.append("&name-pattern=" + URL.encodeQueryString(request.getPattern()));
```

Comment:
Does this still work if the user does not enter any pattern?
I'm not sure if the service will throw an error if the URL query is `... &name-pattern=&folders-only=false ...`